packages/crawler/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "crawler",
+  "version": "0.0.0",
+  "private": true,
+  "license": "UNLICENSED",
+  "scope": "@porsche-design-system",
+  "author": "Dr. Ing. h.c. F. Porsche AG",
+  "scripts": {
+    "start": "ts-node src/index.ts",
+    "test:e2e:puppeteer": "jest --colors -c tests/e2e/config/jest.config.ts",
+    "test:e2e": "concurrently -k -s 'first' -n 'cdn,puppeteer' 'serve-cdn' 'yarn test:e2e:puppeteer'"
+  },
+  "dependencies": {
```

Comment:
For our licensing (OSO) scan the convention is that `dependencies` are dependencies that are bundled/shipped. While this package might be shipped/used in the future we should have this clean already, otherwise everything should be under `devDependencies`.  
- Therefore `typescript` and `ts-node` will never be shipped since those are used at build time.
- `puppeteer` most likely won't be shipped either and is in fact a part of `@porsche-design-system/shared` already and can therefore be removed.
- At the same time `jest` is missing but is provided by `@porsche-design-system/shared`
- `@porsche-design-system/components-js` should have a proper version which should also be updated in the `run-prepare-release-components` bash script.

Reply:
Done